Puppetfile

```diff
@@ -51,7 +51,7 @@ mod 'maestrodev-wget',
   :git => 'https://github.com/maestrodev/puppet-wget',
   :ref => 'master'
 
-mod 'stackstorm-st2',                '0.12.2'
```

Comment:
Looks like a significant jump in this case. I wonder if the changes in between are still kosher. This effectively means a new installer.